data_generators/generators.py

```diff
@@ -52,11 +52,11 @@ def __init__(self):
         self.formerly_incarcerated = percent_true(80)
         self.underrepresented_group = percent_true(70)
         self.low_income = percent_true(70)
-        self.list_convictions = sample(convictions, k=randint(1, 3))
+        self.convictions = sample(convictions, k=randint(1, 3)) 
         self.tech_stack = choice(tech_stack)
         self.job_help = percent_true(33)
         self.pair_programming = percent_true(33)
-        self.heard_about = choice(heard_about_us)
+        self.referred_by = choice(heard_about_us)  
         self.other_info = "anything else may be written here"
```

Comment:
The two whitespaces at the end of this line look repulsive and anti-American. They're everywhere. It's against everything I stand for as a Pep8 Hero to Approve, and yet I must, bc it's just fine otherwise. In fact, it's necessary. Good update, cleaning up old inconsistencies. Code will run now, and run well. Just not beautifully.